corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -709,6 +717,35 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         tagName: 'li',
         childView: LanguageOptionView,
         childViewContainer: 'ul',
+        ui: {
+            dropdownMenu: ".dropdown-menu.dropdown-menu-right",
+            selectPrint: "#print-header.dropdown-header",
+        },
+        childViewOptions: function () {
+            return {
+                languageOptionsEnabled: Boolean(this.options.collection),
+            };
+        },
+        templateContext: function () {
+            var languageOptionsEnabled = Boolean(this.options.collection);
+            return {
+                languageOptionsEnabled: languageOptionsEnabled,
+            };
+        },
+        events: {
+            "keydown": "printKeyAction",
+            "click @ui.selectPrint": "hideDropdownOnPrint",
+        },
+        printKeyAction: function (e) {
+            if (e.keyCode === 13) {
+                this.hideDropdownOnPrint();
+            }
+        },
+        hideDropdownOnPrint: function () {
+            this.ui.dropdownMenu.toggle();
```

Comment:
The demo seems to work fine but curious - why does toggle need to be called twice?

Reply:
if the toggle isn't called again then, then the user is unable to select the dropdown again

Reply:
I see, so toggle enables and disables?  I always thought it only affected visibility

Reply:
I thought so too which is why I just had it once, but then it also disabled the functionality. Also when the second one is called it doesn't make the dropdown appear again. So this makes it disappear and disables, and then reenables but not make it appear again without a click if that makes sense